cmd/flightctl-pam-issuer/main.go

```diff
@@ -0,0 +1,132 @@
+//go:build linux
+
+package main
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/flightctl/flightctl/internal/api_server/middleware"
+	"github.com/flightctl/flightctl/internal/config"
+	"github.com/flightctl/flightctl/internal/crypto"
+	"github.com/flightctl/flightctl/internal/instrumentation"
```

Comment:
```suggestion
	"github.com/flightctl/flightctl/internal/instrumentation/tracing"
```
OTherwise "make deploy" fails.